### DISCUSSION TITLE
Make deployment logs public

### DISCUSSION THREAD

**Discussion by ZeldOcarina** [vercel/community#1433] [Feature Question]
### Summary

I'd like to make some deployment links public so other people outside my team can see what is going on.

I am already using webhooks to change our CRM programmatically and show whether the app is building, cancelled, failed etc. but on every failed build I need to manually go in and see what made it fail.

Is there a way I could get the deployment URL link I get back from the API to be public so they can see themeselves?

I saw the documentation said it's not possible, but maybe you can implement this if it's requested..

**Comment by amyegan** [comment]
You're right @ZeldOcarina, only members of your team have access to view the logs. So a public link isn't currently possible, but I'm going to move this to the "Ideas" category so it can be considered for future addition

**Comment by ZeldOcarina** [comment]
Thanks @amyegan that would be amazing!

**Comment by jaredh159** [comment]
would love to have public logs. netlify supports this.

**Comment by ZeldOcarina** [comment]
btw I fixed this with a complex integration with REST API on Vercel and I've connected the log stream with the CRM the team is using.. so I made the logs "public" that way..

**Reply by jaredh159** [reply]
ok, interesting, thanks for letting me know. 👍

**Comment by leerob** [comment]
If you turn off "logs and source protection", you can make the build logs public.

![CleanShot 2024-03-23 at 19 01 51@2x](https://github.com/vercel/community/assets/9113740/dff949cd-2a2e-4694-904d-815d722cf449)


https://vercel.com/docs/projects/overview#logs-and-source-protection

**Reply by ZeldOcarina** [reply]
That is not an answer as it exposes _src too which is not ideal.
